core/src/main/java/org/mobilitydata/gtfsvalidator/notice/SystemError.java

```diff
@@ -32,8 +32,8 @@
  * because that may leak internal information. Only validation notices should be shared.
  */
 public abstract class SystemError extends Notice {
-  public SystemError(Map<String, Object> context) {
+  public SystemError(Map<String, Object> context, SeverityLevel severityLevel) {
     // by default SystemError.severity is set to SeverityLevel.ERROR
-    super(context, SeverityLevel.ERROR);
```

Comment:
The implementation of `SystemError` was fine as-is and shouldn't change. By definition anything that implements it will be an `ERROR`, so it should be hard-coded as an error here and shouldn't be exposed as a constructor parameter.